plotly/offline/offline.py

```diff
@@ -55,21 +51,13 @@ def init_notebook_mode():
         raise ImportError('`iplot` can only run inside an IPython Notebook.')
     from IPython.display import HTML, display
 
-    if not os.path.exists(_PLOTLY_OFFLINE_BUNDLE):
-        raise PlotlyError('Plotly Offline source file at {source_path} '
-                          'is not found.\n'
-                          'If you have a Plotly Offline license, then try '
-                          'running plotly.offline.download_plotlyjs(url) '
-                          'with a licensed download url.\n'
-                          "Don't have a Plotly Offline license? "
-                          'Contact sales@plot.ly learn more about licensing.\n'
-                          'Questions? support@plot.ly.'
-                          .format(source_path=_PLOTLY_OFFLINE_BUNDLE))
+    if not os.path.exists(PLOTLY_OFFLINE_BUNDLE):
+        raise PlotlyOfflineNotFound
 
     global __PLOTLY_OFFLINE_INITIALIZED
     __PLOTLY_OFFLINE_INITIALIZED = True
     display(HTML('<script type="text/javascript">' +
-                 open(_PLOTLY_OFFLINE_BUNDLE).read() + '</script>'))
+                 open(PLOTLY_OFFLINE_BUNDLE).read() + '</script>'))
 
```

Comment:
@theengineear w.r.t. https://github.com/plotly/python-api/pull/243/files#r33820323, I'm assuming that that `try` `except` is going to catch everything that could possible go wrong in here. I'm what errors `os.path.exists` might bubble up